plugins/updater/src/lib.rs

```diff
@@ -88,6 +93,10 @@ impl<R: Runtime, T: Manager<R>> UpdaterExt<R> for T {
             builder = builder.current_exe_args(args);
         }
 
+        if let Some(version_comparator) = version_comparator {
+            builder = builder.global_version_comparator(version_comparator.clone());
```

Comment:
there is a slight issue with this as the types dont match.
since calling `version_comparator` has a param type of `F: Fn(Version, RemoteRelease) -> bool + Send + Sync + 'static` but the global one has `Arc<dyn Fn(Version, RemoteRelease) -> bool + Send + Sync>` so i have made the field `pub(crate)` but could also use another builder method only in the crate